WAMPClient.js

```diff
@@ -13,6 +12,7 @@ class WAMPClient {
 	}
 
 	static generateSignature(procedureCalls) {
+		/* eslint no-bitwise: ["error", { "int32Hint": true }] */
```

Comment:
tests are showing it is faster but not so drastically
https://jsperf.com/math-round-vs-bitwise-op/1